tests/test-components/PastCommitNode.spec.tsx

```diff
@@ -29,13 +31,15 @@ describe("PastCommitNode", () => {
       is_remote_branch: false,
       name: "name3",
       upstream: "upstream",
+      top_commit: "2414721hijklmnopqrstuvwxyz01234567890123",
```

Comment:
This should be changed to `2414721b194453f058079d897d13c4e377f92dc6` so it matches.

Reply:
Done